src/pages/signin/ChangeExpensifyLoginLink.js

```diff
@@ -33,7 +33,7 @@ const ChangeExpensifyLoginLink = ({credentials, translate, toLocalPhone}) => (
         </Text>
         <TouchableOpacity
             style={[styles.link]}
-            onPress={restartSignin}
+            onPress={() => redirectToSignIn()}
```

Comment:
```suggestion
            onPress={redirectToSignIn}
```

Reply:
If we do that then it can result in error as unnecessary argument will be passed to the function.
 https://github.com/Expensify/App/blob/163746ef7a802819ac879e68626e0170da125989/src/libs/actions/SignInRedirect.js#L34

Reply:
Ah, that's right, I forgot about the event. Good call!